pkg/worker/network.go

```diff
@@ -1,13 +1,449 @@
 package worker
 
 import (
+	"bufio"
+	"context"
 	"errors"
 	"fmt"
+	"log"
 	"net"
 	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/beam-cloud/beta9/pkg/repository"
+	"github.com/coreos/go-iptables/iptables"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/vishvananda/netlink"
+	"github.com/vishvananda/netns"
+	"golang.org/x/sys/unix"
+)
+
+const (
+	containerBridgeLinkName         string        = "br0"
```

Comment:
I think you might be right about this. We should change this to something more specific to this project like br-beta9, or increment the bridge name until we hit a bridge that doesn't exist. I think the former is fine.

Reply:
Done